Content.Server/DeviceNetwork/Systems/StationLimitedNetworkSystem.cs

```diff
@@ -0,0 +1,68 @@
+using Content.Server.DeviceNetwork.Components;
+using Content.Server.Station.Systems;
+using JetBrains.Annotations;
+using Robust.Shared.Map;
+
+namespace Content.Server.DeviceNetwork.Systems
+{
+    /// <summary>
+    /// This system requires the StationLimitedNetworkComponent to be on the the sending entity as well as the receiving entity
+    /// </summary>
+    [UsedImplicitly]
+    public sealed class StationLimitedNetworkSystem : EntitySystem
+    {
+        [Dependency] private readonly StationSystem? _stationSystem = default!;
```

Comment:
```suggestion
        [Dependency] private readonly StationSystem _stationSystem = default!;
```
will always be valid for entity systems